zone/npc.cpp

```diff
@@ -2901,15 +2901,15 @@ void NPC::DoNPCEmote(uint8 event_, uint32 emote_id, Mob* t)
 	// Mob Variables
 	Strings::FindReplace(processed, "$mname", GetCleanName());
 	Strings::FindReplace(processed, "$mracep", GetRacePlural());
-	Strings::FindReplace(processed, "$mrace", GetPlayerRaceName(GetRace()));
+	Strings::FindReplace(processed, "$mrace", GetRaceIDName(GetRace()));
```

Comment:
Unsure about the change to RaceID vs. just Race

Reply:
We had a separate method that used `GetRaceIDName` under the hood just for player races, so it was removed.